src/main/java/network/brightspots/rcv/Main.java

```diff
@@ -73,12 +73,15 @@ public static void main(String[] args) {
             argsCli.size());
         System.exit(2);
       }
-      // config file for configuring the tabulator
-      String configPath = argsCli.get(0);
-      boolean convertToCdf = argsCli.size() == 2 && argsCli.get(1).equals("convert-to-cdf");
-      // session object will manage the tabulation process
-      TabulatorSession session = new TabulatorSession(configPath);
-      if (convertToCdf) {
+      // Path to either: config file for configuring the tabulator, or Dominion JSONs
+      String providedPath = argsCli.get(0);
+      if (argsCli.size() == 2 && argsCli.get(1).equals("convert-dominion-to-generic-csv")) {
+        TabulatorSession.convertDominionCvrJsonToGenericCsv(providedPath);
+        System.exit(0);
```

Comment:
I don't feel strongly about this, but I think it would be a bit cleaner to combine this with the if block below and just have a single `System.exit` call at the end of the method.

Reply:
Sure, I guess it's slightly more readable that way. I was trying to avoid creating `TabulatorSession session = new TabulatorSession(providedPath);` without needing to use it, but no big deal.